### PR TITLE
Adding check for prefix usage with "source" and "hasDbXref" properties

### LINKED ISSUE #4597: Major prefix cleanup
To prepare for our upcoming syncing efforts, I finally got around to do some major cleanups of the usage of prefixes in Mondo.

#4595 

I am have cleaned up quite a bit, but I need some help figuring out what to do with these:

- [x] CSP2005
- [x] MTHICD9_2006
- [x] ORDO
- [x] MONDOLEX
- [ ] MONDORULE
- [x] OWLReasoner
- [ ] DC-OMIM
- [x] ISBN-13
- [x] NCBI
- [x] NCI2004_11_17
- [x] Rename ICD10 to ICD10CM
- [ ] NIFSTD (should be removed)
- [ ] MTH (should be removed)

### PR DESCRIPTION
**Pull request by matentzn**
fixes #4597

### CODE REVIEW

File: src/sparql/qc/mondo/qc-illegal-prefix-on-xref-annotation.sparql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ?axiom owl:annotatedSource ?cls ;
+           owl:annotatedProperty ?mapping ;
+           owl:annotatedTarget ?value ;
+           ?property ?source .
+    
+  
+  FILTER NOT EXISTS {
+  	?cls owl:deprecated ?deprecated .
+  }
+  FILTER( STRBEFORE(str(?source),":") not in ("UMLS", "DOID", "OMIM", "Orphanet", "SCTID", "MESH", "NCIT", "GARD", "ICD9", "EFO", "MedDRA", "ICDO", "HP", "ONCOTREE", "OMIMPS", "Wikipedia", "GTR", "HGNC", "CSP", "KEGG", "PMID", "MEDGEN", "NIFSTD", "MONDO", "ICD10CM", "OMOP", "MFOMD", "MP", "OGMS", "Wikidata", "SCDO", "ICD11", "NDFRT", "LOINC", "IDO", "PATO", "OBI", "MTH", "Reactome", "DERMO", "MPATH", "ICD9CM", "ISBN", "doi", "MEDIC", "ClinGen", "HPO", "ECO", "MedGen", "OMIT"))
```
Comment (cthoyt) [inline src/sparql/qc/mondo/qc-illegal-prefix-on-xref-annotation.sparql:31]: I would suggest splitting this list on to multiple lines so it's possible to comment on individual ones, and so it's easier to read future diffs that might change it.

**Comment by matentzn** [comment]
- [ ] @matentzn to address @cthoyt comment again

File: src/sparql/qc/mondo/qc-illegal-prefix-on-xref-annotation.sparql (hunk 1 above)
Comment (matentzn) [reply src/sparql/qc/mondo/qc-illegal-prefix-on-xref-annotation.sparql]: Done

**Comment by matentzn** [comment]
We are getting closer did another chunk today #4739 next week more

**Comment by matentzn** [comment]
@nicolevasilevsky this is a nice issue to close soon, just 61 violations left which we could start in 10 minutes together, and you can finish them in another 30?

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**